src/xmipp/applications/scripts/deepRes_resolution/batch_deepRes_resolution.py

```diff
@@ -202,8 +210,29 @@ def produceOutput(fnVolIn, fnMask, model, sampling, Y, fnVolOut):
                           V[z,y,x]=meansum         
 
 
-    Vxmipp.setData(V)
-    Vxmipp.write(fnVolOut)
+    if fnVolOut is not None:
+      Vxmipp = xmippLib.Image()
+      Vxmipp.setData(V)
+      Vxmipp.write(fnVolOut)
+    return V
+
+def main(fnModel, fnVolIn, fnMask, sampling, fnVolOut):
+
+  if fnModel=="default":
+    fnModel= XmippScript.getModel("deepRes", "model_w13.h5")
```

Comment:
these names are correct? Maybe some global constant would be better

Reply:
Well, this is the way we have been doing for one year. Constants are always better of course.

Reply:
We could work on this, but not now. I will note down and do it in a near future.